pytorch/jobs/docker/1.9/py3/Dockerfile.gpu

```diff
@@ -1,7 +1,7 @@
 ﻿# This is based loosely on :
 # https://github.com/aws/amazon-sagemaker-examples/blob/master/advanced_functionality/custom-training-containers/script-mode-container/docker/Dockerfile
 
-FROM public.ecr.aws/ubuntu/ubuntu:18.04
+FROM 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training:1.9.1-gpu-py38-cu111-ubuntu20.04
```

Comment:
If we're getting the base image from DLC, then we don't need the majority of the functionality of this docker file. It should be as small as the TF one.